.travis.yml

```diff
@@ -24,6 +23,9 @@ jobs:
     - stage: linting
       name: "pre-commit"
       before_install:
+        - sudo add-apt-repository -y "deb https://apt-archive.postgresql.org/pub/repos/apt trusty-pgdg-archive main"
```

Comment:
about this : postgis doesn't seem to be installable on trusty anymore.
https://www.postgresql.org/message-id/YBMtd6nRuXyU2zS4%40msg.df7cb.de